kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/jdbc/JDBCMetadataStoreConf.scala

```diff
@@ -93,4 +93,16 @@ object JDBCMetadataStoreConf {
       .serverOnly
       .stringConf
       .createWithDefault("")
+
+  val METADATA_STORE_JDBC_PRIORITY_ENABLED: ConfigEntry[Boolean] =
+    buildConf("kyuubi.metadata.store.jdbc.priority.enabled")
+      .doc("Whether to enable picking up jobs from the metadata store in order of priority." +
+        "If this feature is disabled, setting kyuubi.batch.priority will not take effect." +
+        "However, it is recommended to enable this feature if you already have a metadata store " +
+        "that can support mixed order indexing(such as MySQL 8)." +
+        "See why recommend this in https://github.com/apache/kyuubi/pull/5329")
```

Comment:
suggested change:

Whether to enable the priority scheduling for batch impl v2. When false, ignore `kyuubi.batch.priority` and use the FIFO ordering strategy for batch job scheduling. Note: this feature may cause significant performance issues when using MySQL 5.7 as the metastore backend due to the lack of support for mixed order index. See more details at KYUUBI #5329.

Also, please take care of the space between each line.